invenio/modules/knowledge/models.py

```diff
@@ -166,6 +185,62 @@ def set_dyn_config(self, field, expression, collection=None):
                                     search_expression=expression,
                                     collection=collection)
 
+    @staticmethod
+    def generate_slug(name, tries=10):
```

Comment:
@lnielsen What do you think is better?
1. Use a function like this to generate the slug.
2. Put a restriction on the name (valid names are only `[:alnum:]`)
3. Any better idea? :)